package.json

```diff
@@ -36,6 +36,7 @@
     "bitcore-message": "github:CoMakery/bitcore-message#dist",
     "bs58": "^4.0.1",
     "chloride": "^2.2.8",
+    "crypto-ld": "digitalbazaar/crypto-ld#signer",
```

Comment:
This also needs to use the new crypto-ld package, not a branch.

Reply:
Fixed.